staging/provisioning/windows/windowsnodereset.ps1

```diff
@@ -49,6 +49,11 @@ if ($global:EnableHostsConfigAgent) {
 # Perform cleanup
 #
 
+Write-Log "Cleaning up persisted HNS policy lists"
+# Workaround for https://github.com/kubernetes/kubernetes/pull/68923 in < 1.14,
+# and https://github.com/kubernetes/kubernetes/pull/78612 for <= 1.15
```

Comment:
Since this "workaround" is still lets update the comments again to point the behavior you found in kube-proxy.